doc/Doxyfile

```diff
@@ -35,13 +35,13 @@ ENABLE_PREPROCESSING = NO
 HTML_COLORSTYLE   = LIGHT
 CASE_SENSE_NAMES  = NO
 IMAGE_PATH        = . images
-INPUT             = $(INDEX_DOC) install.doc starting.doc docblocks.doc additional_documentation.dox \ 
-					markdown.doc lists.doc grouping.doc formulas.doc tables.doc diagrams.doc preprocessing.doc \
-                    autolink.doc output.doc searching.doc extsearch.doc customize.doc custcmd.doc \
-                    external.doc faq.doc trouble.doc features.doc doxygen_usage.doc doxywizard_usage.doc \
-					config.doc commands.doc htmlcmds.doc xmlcmds.doc emojisup.doc language.doc \
-                    perlmod.doc arch.doc changelog.doc 
-FILE_PATTERNS     = *.cpp *.h *.doc *.dox
+INPUT             = $(INDEX_DOC) install.dox starting.dox doxblocks.dox additional_doxumentation.dox \ 
```

Comment:
I think the search/replace was a bit too eager: the build fails because of do**x**block.dox and additional_do**x**umentation.dox not found.